### ISSUE TITLE
In Skype 8 for desktop, NVDA does not read automatically outgoing chat messages

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#8591]
### Steps to reproduce:
1. Install Skype 8 for Desktop.
2. Open a chat with somebody.
3. Type a message and press ENTER to send it.

### Actual behavior:
NVDA only reports "Message sent".

### Expected behavior:
NVDA should also automatically read the outgoing (sent) message, as it reads automatically the incoming ones in the current chat.

### System configuration:
- Operating system: Windows 10 Pro version 1803 (build 17134.191), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- Skype version: 8.28.76.19 Preview, but the issue is reproducible with the stable 8.27.0.85 as well.
- NVDA version: alpha-15781,d58e1cd4, installed.
- Processor: Intel Core i5-4670 at 3.40GHz.
- RAM Memory: 8.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory + 4065MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC887 at Intel Lynx Point PCH - High Definition Audio Controller.

#### Does the issue still occur after restarting your PC?
Yes, it does.

#### Have you tried any other versions of NVDA?
No, I haven't.

**Comment by k-kolev1985** [comment]
@Adriani90 Yes, the issue is still reproducible. I've tried it even with add-ons disabled - it's the same. I have the Windows 10 App Essentials add-on from Joseph Lee, but that seams not to help either.

### Test environment:
- Operating system: Windows 10 Pro version 1809 (build 17763.253), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- Skype 8 for desktop version: 8.37.76.92 Preview.
- NVDA version: alpha-16530,d5120846.
- Processor: Intel Core i5-4670 at 3.40GHz.
- RAM Memory: 8.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory + 4065MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC887 at Intel Lynx Point PCH - High Definition Audio Controller.

**Comment by Adriani90** [comment]
@k-kolev1985 after giving this a thought, I think it doesn't make sense to read the outgoing message in full. Imagine a very long message you send to some one, actually you don't want to hear it again unless you navigate through the message list. I think the notification that the message you typed has been sent is the appropriate behavior here. I am closing this one. Please comment on this in case you have a strong argument why it makes sense to hear the outgoing message again after pressing enter, and we can reopen this then if appropriate.
